packages/generator/generator-common/src/locale/en.ts

```diff
@@ -27,10 +27,8 @@ export const EN_LOCALE = {
       i18n: 'Enable Internationalization (i18n)',
       test: 'Enable Unit Test / Integration Test',
       e2e_test: 'Enable E2E Test',
-      storybook: 'Enable Storybook',
       storybookV7: 'Enable Storybook V7',
```

Comment:
Should we rename the 'Enable Storybook V7' to 'Enable Storybook'?

Reply:
In legacy generator 'Enable Storybook' means enable Storybook v6, I'm afraid if people read the new document and use legacy generator, they enable Storybook v6 by mistake.

Reply:
get